README.md

```diff
@@ -13,8 +16,25 @@ If you don't feel like typing citations out (and let's be honest, you don't), ex
 - Activate via Command Palette (command + shift + P): Type "Zotero Citation Picker" and press enter.
 - Activate via keyboard shortcut: Use alt+shift+z
 
-![Screenshot](images/screenshot.png)
+<img src="images/screenshot.png" style="width: 80%;" alt="Screenshot">
 
 ## Requirements
 
 **IMPORTANT:** Zotero, with Better BibTeX installed, must be running while you use these.
+
+## Configuration
+
+The formats which are supported by this plugin depend on the [CAYW (Cite as you Write)](https://retorque.re/zotero-better-bibtex/citing/cayw/index.html) implementation by zotero-better-bibtex.
+
+An excerpt of the formats that are supported are as follows:
+- `cite` for latex, uses the cite command
```

Comment:
`cite` is an alias for `natbib`, where the default (but configurable) command is `cite`